### DISCUSSION TITLE
Ctrl v instead of Command v

### DISCUSSION THREAD

**Discussion by MoAbbasid** [p0deje/Maccy#1002] [Q&A] [answered]
Hi, thanks for making the app so minimal and simple,
one thing Im wondering is that I have my copy paste to ctrl c and ctrl v respectively, ctrl c works as expected however i noticed that with maccy you can not paste directly from the menu using Cmd1, you have to use cmd to select then ctrl v to paste, 
Im wondering if there anyway that I can make that one click to select and paste, even if its ctrl v for the normal paste and cmd1, for maccy paste
Thank you

**Comment by p0deje** [comment]
Have you enabled "Paste automatically"?

**Reply by MoAbbasid** [reply]
Yes but it still requires two presses, 1. Cmd+1, 2.Ctrl+v

**Reply by p0deje** [reply, marked as answer]
Can you check that Maccy is added to System Settings / Privacy & Security / Accessibility? If not, please add it by clicking on "+" button and selecting Maccy.

**Reply by MoAbbasid** [reply]
it is working now, Thank you very much for your assistance,
one other question, can I make it one click to paste with the cmd+numbers without even having to open the popup?

**Reply by p0deje** [reply]
No, it's not supported at the moment.
